packages/react/src/SlashMenu/components/SlashMenuPositioner.tsx

```diff
@@ -36,21 +31,15 @@ export const SlashMenuPositioner = <
     useState<number>();
 
   const referencePos = useRef<DOMRect>();
-  const callbacks =
-    useRef<SuggestionsMenuCallbacks<ReactSlashMenuItem<BSchema>>>();
 
   useEffect(() => {
-    callbacks.current = props.editor.createSlashMenu((slashMenuState) => {
+    return props.editor.slashMenu.on("update", (slashMenuState) => {
```

Comment:
Is there any point having the first param? Isn't it always just "update"? Same for the other UI elements